shared/teams/main/banner.tsx

```diff
@@ -56,16 +59,17 @@ const styles = Styles.styleSheetCreate({
       alignItems: 'center',
       backgroundColor: Styles.globalColors.blue,
       flexShrink: 0,
-      justifyContent: 'center',
       position: 'relative',
       width: '100%',
     },
     isElectron: {
       ...Styles.globalStyles.flexBoxRow,
       height: 212,
+      justifyContent: 'flex-start',
```

Comment:
Want to add some `paddingRight` to this?
<img width="335" alt="image" src="https://user-images.githubusercontent.com/11968340/59924559-fce21b00-9403-11e9-90dd-0fe26e71801b.png">